Source/Core/DolphinQt/Debugger/WatchWidget.cpp

```diff
@@ -111,6 +118,10 @@ void WatchWidget::ConnectWidgets()
 
 void WatchWidget::UpdateIcons()
 {
+  // TODO: Create a "debugger_add_watch" icon
```

Comment:
Who would create this?  How do we keep track of this other than a code "TODO"?  Maybe an issue in the issue tracker?  I have some icons needed too..

Reply:
@MayImilae would be the person to talk to.

Reply:
@JosJuice - is there a way we can track this or must we block this PR until MayImilae has time to create said icons?

Reply:
I don't think we really have a standard way to track icon requests... But I suppose the issue tracker could make sense.